modules/configuring-the-distributed-workloads-components.adoc

```diff
@@ -0,0 +1,194 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-distributed-workloads-components_{context}"]
+= Configuring the distributed workloads components
+
+[role='_abstract']
+To configure the distributed workloads feature for your data scientists to use in {productname-short}, you must enable several components.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+* You have access to the data science cluster.
+* You have installed {productname-long}.
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.6 vCPU and 2 GiB memory to deploy the distributed workloads infrastructure.
+* You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/cluster/getting-started.html[Ray Clusters documentation].
++
+[NOTE]
+====
+Mutual Transport Layer Security (mTLS) is enabled by default in the CodeFlare component in {productname-short}.
+In the current {productname-short} version, `submissionMode=K8sJobMode` is not supported in the Ray job specification, so the KubeRay Operator cannot create a submitter Kubernetes Job to submit the Ray job.
+Instead, the Ray job specification must be configured to set `submissionMode=HTTPMode` only, so that the KubeRay Operator sends a request to the RayCluster to create a Ray job.
+====
+* You have access to the data sets and models to be used by the distributed workload.
```

Comment:
You have access to the data sets and models that you want the distributed workload to use (or to process?)